glossary.yml

```
@@ -1064,6 +1064,11 @@
       The file(s) that a [build rule](#build_rule) will update if they are out-of-date
       compared to their [dependencies](#dependency).
 
+- slug: byte
+  en:
+    term: "byte"
+    def: >
+      A unit of digital information that typically consists of eight binary digits, or bits.
```

```suggestion
      A unit of digital information that typically consists of eight [binary](#binary) digits, or [bits](#bit).
```
I've suggested some internal links to other entries. The definition looks good to me.